### PR TITLE
updated function comments

### PR DESCRIPTION
**Pull request by bpbirch**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
updated code / function signature comments

* **What is the current behavior?** (You can also link to an open issue here)
nonexistent or out of date comments

* **What is the new behavior (if this is a feature change)?**
existent and up to date comments

* **Other information**:

### CODE REVIEW

File: src/commands/export.ts [hunk 1]
```diff
@@ -58,7 +62,13 @@ export const handler = async (args: yargs.Arguments): Promise<any> => {
     return Promise.resolve(null);
 };
 
-
+/**
+ * export contract to simbachain.com (can also be thought of as "importing" contract to simbachain.com)
+ * @param interactive - export from prompts if true
+ * @param primary - optional param specifying which contract to export. if not present, contract is selected from prompts
+ * @param savemode - if 'new' we do a post request for new design design; if 'update' we do a put request and update contract design
```
Comment (JehoG) [inline src/commands/export.ts:69]: design design?

File: src/commands/getdirs.ts [hunk 2]
```diff
@@ -7,13 +7,24 @@ export const command = 'getdirs';
 export const describe = 'get paths for project-relevant directories';
 export const builder = {};
 
+/**
+ * view relevant directories (contracts, build, etc.)
```
Comment (JehoG) [inline src/commands/getdirs.ts:11]: Is this the good doc? Same as lower but for "handler"?

File: src/commands/login.ts [hunk 3]
```diff
@@ -46,6 +49,17 @@ export const handler = async (args: yargs.Arguments): Promise<any> => {
...
 };
 
+/**
+ * command for user to login (acquire auth token, and choose org and app from blocks)
+ * @param interactive - if true, then choose org and app from prompts
+ * Also, if true, auth token will be generated through device login flow
+ * If false, auth token will be generated through client creds flow,
+ * so SIMBA_AUTH_CLIENT_ID and SIMBA_AUTH_CLIENT_SECRET need to be present
+ * in .simbachain.env, simbachain.env, or .env in project root or SIMBA_HOME
+ * @param org - if !interactive, org must be specified, or be present in simba.json
```
Comment (JehoG) [inline src/commands/login.ts:59]: Maybe if false or null?

File: src/commands/resetdir.ts [hunk 4]
```diff
@@ -16,6 +16,13 @@ export const builder = {
     },
 };
 
+/**
+ * reset directory to default directory (contracts/, buid/, etc.)
```
Comment (JehoG) [inline src/commands/resetdir.ts:20]: typo: buid

File: src/commands/resetdir.ts [hunk 5]
```diff
@@ -29,6 +36,11 @@ export const handler = (args: yargs.Arguments): any => {
     return;
 };
 
+/**
+ * reset directory to default directory (contracts/, buid/, etc.)
```
Comment (JehoG) [inline src/commands/resetdir.ts:40]: Same typo: buid

File: src/commands/getdirs.ts (hunk 2 above)
Comment (bpbirch) [reply src/commands/getdirs.ts:11]: I followed the same pattern of just slightly modifying comments for handler, and the function that the handler calls. 

I'm not sure I understand your question here.

**Review by JehoG — APPROVED** [review]

**Pull request merged**